src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -564,17 +599,35 @@ private Void logMessageOnBatchCompletion(final BatchCompletionCallbackFunctionAr
         }).collect(Collectors.toList());
     }
 
+    private void generateIntervalListFromWorkspace() {
```

Comment:
It would be really nice to be able to call just this method from the command line as like a dry-run mode or staging or something.